Makefile

```diff
@@ -17,12 +19,14 @@ build:
 
 .PHONY: docker-build
 docker-build:
-	docker build -t $(DOCKER_IMAGE_REPO):$(SLO_EXPORTER_VERSION) .
-	docker run --rm $(DOCKER_IMAGE_REPO):$(SLO_EXPORTER_VERSION) --help
+	docker build -t $(DOCKER_IMAGE) .
+	docker run --rm $(DOCKER_IMAGE) --help
 
 .PHONY: docker-push
 docker-push:
-	docker push $(DOCKER_IMAGE_REPO):$(SLO_EXPORTER_VERSION)
+	docker push $(DOCKER_IMAGE)
+	docker tag $(DOCKER_IMAGE) $(LATEST_DOCKER_IMAGE)
```

Comment:
I'd suggest renaming variable `LATEST_DOCKER_IMAGE` to `DOCKER_IMAGE_LATEST`

Reply:
This means we will not be able to push fix of any of older versions. It's OK for now, but later this might be an issue.